jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -709,7 +712,7 @@ public InvocationType getInvocationType()
             return _callback.getInvocationType();
         }
 
-        private boolean reset(MetaData.Request request, MetaData.Response info, ByteBuffer content, boolean last, Callback callback)
+        private boolean reset(MetaData.Request request, MetaData.Response info, ByteBuffer[] content, boolean last, Callback callback)
```

Comment:
As we are changing the signature of this method, let's change it's name also (as I always thought it was not right).  How about `prepare` ?

Reply:
Done, this is a private method anyway is no problem changing it.